entities/api/main.py

```diff
@@ -55,6 +56,8 @@ def get_app() -> FastAPI:
     Make sure tests can get their own instances of the app.
     """
     settings = APISettings.model_validate({})  # Workaround for https://github.com/pydantic/pydantic/issues/3753
+    File.set_settings(settings)
+    File.set_s3_client(get_s3_client(settings))
```

Comment:
heads-up that this s3 client will be a *different client* than the client returned by a FastAPI dependency resolution call. FastAPI memoizes dependency results, and a random call to `get_s3_client()` won't. I don't think this is likely to be a problem since all clients invoked with the same settings should be equivalent, and there's not a lot of state in these clients.